src/main/java/net/javadiscord/javabot2/config/guild/ModerationConfig.java

```diff
@@ -16,6 +16,31 @@
 	 */
 	private long staffRoleId;
 
+	/**
+	 * The amount of seconds to be looked into the past to determine if a user is spamming.
+	 */
+	private int pastMessageCountBeforeDurationInSeconds;
+
+	/**
+	 * The amount of messages to be sent within {@link #pastMessageCountBeforeDurationInSeconds}.
+	 */
+	private int messageSpamAmount;
+
+	/**
+	 * The amount of messages to be cached for each user.
+	 */
+	private int cachedMessagesPerUser;
+
+	/**
+	 * The frequency of cleaning up the cached messages. Amount in minutes.
+	 */
+	private int cachedMessageCleanupFrequency;
+
+	/**
+	 * The amount of minutes for removing this cached messages.
+	 */
+	private int amountOfMinutesForRemoval;
+
```

Comment:
Same thing for these; include `spam` as a prefix to the variable names for clarity. Also include time units in the variable name so it is clear to users who are editing the config JSON directly.